examples/platform/linux/AppMain.cpp

```diff
@@ -369,6 +369,71 @@ class SampleTestEventTriggerHandler : public TestEventTriggerHandler
     }
 };
 
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+void int_array_add_unique(uint16_t ** res, uint16_t a)
+{
+    size_t reslen, max_size;
+    uint16_t * n;
```

Comment:
move this closer to usage instead of at a top. Also give it a better name than `n`. What does it mean.

TLDR: I cannot tell at a glance what this method is doing and what its logic or variables mean. Please make this readable.

Reply:
This is actually extracted from wpa_supplicant source code which processes the similar function. I will rewrite it